setup.py

```diff
@@ -606,13 +606,17 @@ def get_package_dirs():
             # Install the contents of each backend's `language` directory into
             # `triton.language.extra`.
             for x in os.listdir(backend.language_dir):
-                yield (f"triton.language.extra.{x}", os.path.join(backend.language_dir, x))
+                full_path = os.path.join(backend.language_dir, x)
+                if os.path.isdir(full_path):
+                    yield (f"triton.language.extra.{x}", os.path.join(backend.language_dir, x))
 
         if backend.tools_dir:
             # Install the contents of each backend's `tools` directory into
             # `triton.tools.extra`.
             for x in os.listdir(backend.tools_dir):
-                yield (f"triton.tools.extra.{x}", os.path.join(backend.tools_dir, x))
+                full_path = os.path.join(backend.tools_dir, x)
+                if os.path.isdir(full_path):
+                    yield (f"triton.tools.extra.{x}", os.path.join(backend.tools_dir, x))
```

Comment:
i think we can reuse the `full_path` variable in the yield statement. same for above.